File: src/main/java/com/github/thorben/webnd/application/controller/api/dto/SimpleMessages.java
package com.github.thorben.webnd.application.controller.api.dto;

import java.util.ArrayList;
import java.util.List;

public class SimpleMessages {

	private final List<String> messages = new ArrayList<>();

	public SimpleMessages(String... msg) {
		List.of(msg).forEach(this::addMessage);
	}

	public void addMessage(String message) {
		messages.add("{\"msg\":\"" + message + "\"}");
	}

	public boolean filled() {
		return !messages.isEmpty();
	}

}

File: src/main/kotlin/com/github/thorben/webnd/application/controller/api/dto/SimpleMessages.kt
package com.github.thorben.webnd.application.controller.api.dto

import java.util.*
import java.util.function.Consumer

class SimpleMessages(vararg msg: String) {

	init {
		listOf(*msg).forEach { addMessage(it) }
	}

	private val messages: MutableList<String> = ArrayList()

	fun addMessage(message: String) {
		messages.add("{\"msg\":\"$message\"}")
	}

	fun filled(): Boolean {
		return messages.isNotEmpty()
	}
}